nestkernel/source_table.h

```diff
@@ -119,11 +120,14 @@ class SourceTable
   bool previous_entry_has_same_source_( const SourceTablePosition& current_position,
     const Source& current_source ) const;
 
-  void populate_target_data_fields_( const SourceTablePosition& current_position,
+  bool populate_target_data_fields_( const SourceTablePosition& current_position,
     const Source& current_source,
     const thread source_rank,
     TargetData& next_target_data ) const;
 
+  std::vector< std::vector< std::map< index, SpikeData > > > compressible_sources_;
```

Comment:
Add doxygen comments describing purpose and structure of the two new data structures.

Reply:
ok